gaming-website/src/javaScript/games.json

```diff
@@ -0,0 +1,184 @@
+{
+  "payload": {
+    "gamesCover": [
```

Comment:
Not a fan of the structure. 
You have the name of the game repeating itself in multiple places.

What I think you did here is structure the json they way your page is structured or something like that. 

If the design changes, will the json file change as well? It shouldn't. 

So please change the structure. 
Each game should have one entry with all the data.
Then you change your logic to select only what is of interest in your sections